README.md

```diff
@@ -126,6 +126,12 @@ For building Arrow (including Parquet) and its dependencies, we recommend using
 > ./build_linux.sh
 > dotnet build csharp.test --configuration=Release
 ```
+**macOS (unsupported)**
```

Comment:
Not sure if there is any value to claim it as unsupported at this point, given that it's CIed as everything else. Maybe Experimental would be a better warning?

My main worry is more around future upgrades of Arrow, and whether the MacOOS branch is going to delay a release... We'll see

Reply:
Also, MacOS rather than macOS for consistency?

Reply:
Good point too, I went with experimental at first, and then changed it to unsupported for that exact reason. It’s not experimental, as the unit tests work. And it is unsupported, as we don’t support it. Maybe alpha or beta?

For the name, the official one is macOS (don’t shoot the messenger!)

Reply:
Maybe nothing at all then? It'll all depend whether macOS gets in the way of future releases for GR or not really.

macOS instead of MacOS, you're right I've just checked Wikipedia. o_O